apps/nrm/util/transform-data.js

```diff
@@ -1,74 +0,0 @@
-'use strict';
-const moment = require('moment');
```

Comment:
`moment` doesn't seem to be used anywhere else; can we remove this as a dependency entirely?

Reply:
It's still in the package-lock.json